### PR TITLE
fix error in dashboard-mobile and color for sass

### PR DESCRIPTION
**Pull request by Adonisrq94**
@jfavellar90 
@diegomillan 
@nicovanniekerk 
change responsive in dashboard 
fix style erros for all links
change link colors for rocket chat

### CODE REVIEW

File: edx-platform/pearson-theme/lms/static/sass/partials/lms/theme/_dashboard.scss [hunk 1]
```diff
@@ -81,30 +81,44 @@ div.my-courses div.empty-dashboard-message {
     width: 100%;
   }
 
+  /*
```
Comment (jfavellar90) [inline edx-platform/pearson-theme/lms/static/sass/partials/lms/theme/_dashboard.scss]: can we delete this commented block?

File: edx-platform/pearson-theme/lms/static/sass/partials/lms/theme/_dashboard.scss [hunk 2]
```diff
@@ -81,30 +81,44 @@ div.my-courses div.empty-dashboard-message {
...
+
+      a,span {
+        display: inline-block;
+        margin-bottom: 10px;
+        text-decoration: none;
+      }
+    }
   }
 
+  /*
```
Comment (jfavellar90) [inline edx-platform/pearson-theme/lms/static/sass/partials/lms/theme/_dashboard.scss]: can we delete this commented block?

**Comment by jfavellar90** [comment]
@Adonisrq94 just did a couple of comments. Please review the comments so we can merge this to master

**Comment by nicovanniekerk** [comment]
@Adonisrq94 i agree with @jfavellar90  please remove the comment blocks.

File: edx-platform/pearson-theme/lms/static/sass/partials/lms/theme/_dashboard.scss [hunk 3]
```diff
@@ -81,29 +81,26 @@ div.my-courses div.empty-dashboard-message {
     width: 100%;
   }
 
-  .course .details .wrapper-course-details {
+  .course .details .wrapper-course-details  {
```
Comment (jfavellar90) [inline edx-platform/pearson-theme/lms/static/sass/partials/lms/theme/_dashboard.scss]: I see in the code sometimes you use 0,1 or two spaces between the selector and the curly braces, can we use a standard, leaving only one space? This is to have an uniform code

File: edx-platform/pearson-theme/lms/static/sass/partials/lms/theme/_dashboard.scss [hunk 4]
```diff
@@ -81,29 +81,26 @@ div.my-courses div.empty-dashboard-message {
...
-    padding-top: 10px;
-  }
 
-  .course .details .wrapper-course-details .course-title a,
-  .course .details .wrapper-course-details .course-title span {
-    display: inline-block;
-    margin-bottom: 10px;
-    text-decoration: none;
-  }
+    .course-title{
```
Comment (jfavellar90) [inline edx-platform/pearson-theme/lms/static/sass/partials/lms/theme/_dashboard.scss]: Here for example you left no space

**Pull request merged**